sklearn/tests/test_pipeline.py

```diff
@@ -392,6 +393,22 @@ def test_pipeline_methods_preprocessing_svm():
 
         pipe.score(X, y)
 
+def test_pipeline_methods_pca_tsne():
```

Comment:
You need one more blank line above (two blank lines instead of one).

Reply:
Pushed a fix